.travis.yml

```diff
@@ -0,0 +1,40 @@
+# Tell Travis CI we're using PHP
+language: php
+
+# Opt to use Travis container-based environment.
+sudo: false
+
+# Newer versions like trusty don't have PHP 5.2 or 5.3
+# https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming
+dist: precise
```

Comment:
With the work we're doing to bump minimum version, let's code towards a `7.x` future and only test down to 5.6 at minimum.

Reply:
We can test down to 5.6, which is fine, but we still would support 5.3 through static analysis.

Reply:
I slightly disagree, 5.3 is long past EOL and the timeline we have for this would put us into a zone where we shouldn't have to worry about it with regard to core development. I don't think it should hold this up but I don't think we need to go all the way back to precise.